src/mlpack/bindings/python/setup_readme.md

```diff
@@ -0,0 +1,5 @@
+mlpack is a fast, flexible machine learning library, written in C++, that aims
+to provide fast, extensible implementations of cutting-edge machine learning
+algorithms.  mlpack provides these algorithms as simple command-line programs,
+Python bindings, and C++ classes which can then be integrated into larger-scale
+machine learning solutions.
```

Comment:
@rcurtin From the perspective of a Python user, this description might be too C++ focused.  Such users don't really care what language this software was written in, as long as it works in Python (and hence implicitly provides a Python interface).

The second sentence is the main problem here.  Someone using pypi is already getting the Python bindings to mlpack, so the sentence can be considered as redundant.  The part about C++ and bindings may also be confusing -- a user may think they need to compile the bindings or need to do extra steps, etc.

Overall I suggest to simplify the description so that it addresses the target audience, ie. Python users that just want a bunch of ML algorithms working on their data.  If they want to dig further they can always go to the mlpack website.

Reply:
Yeah, you are right.  I revamped the README and put my back into it a little bit this time in 35a85716fa.